web/client/components/map/openlayers/Map.jsx

```diff
@@ -237,6 +243,10 @@ class OpenlayersMap extends React.Component {
     }
 
     UNSAFE_componentWillReceiveProps(newProps) {
+        const elements = document.querySelectorAll(':hover');
+        if (this.props.mousePosition && !this.props.isMouseMoveOverTools && !Array.from( elements ).some(el => el.id === 'map')) {
```

Comment:
you should intercept onmouseout (also the method and the event should be named "onMouseOut").
For OL it should be something like:
map.getViewport().addEventListener('mouseout')

Reply:
Thanks. Done.